ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -221,17 +227,18 @@ snpe_subplugin::runtimeToString (zdl::DlSystem::Runtime_t runtime)
  * @brief Internal method to get names of output layers from tensors information.
  */
 bool
-snpe_subplugin::set_output_layer_names (const GstTensorsInfo *info)
+snpe_subplugin::set_output_tensor_names (const GstTensorsInfo *info)
 {
+  if (output_tensor_names_list.size () > 0) {
+    output_tensor_names_list = zdl::DlSystem::StringList ();
```

Comment:
Isn't this a memory leak? or SNPE manages it?

(the previous contents of output_tensor_names_list is dangled? don't you need ```delete output_tensor_names_list``` ?)

Reply:
This line mean to initialize the list. SNPE manages the destruction.

Reply:
Ok!